zip-1012.rst

```diff
@@ -213,34 +215,39 @@ following additional constraints:
    ecosystem; this is more restrictive than ZF's general mission of furthering
    financial privacy.
 
-6. Major Grants awarding is subject to individual approval by ZF's Board of
-   Directors, by a majority excluding any members with a conflict of interest.
-
-7. ZF shall seek advisory input on its choice of Major Grant awards, by all
-   effective and reasonable means (e.g., on-line discussion forums, the
-   community Advisory Board, on-chain voting by holders and miners, and
-   proactive consultation with experts). The ZF Board of Directors shall
-   strive to follow this advisory input (within the confines of the
-   Foundation's charter and duties).
-
-8. ZF shall strive to create an independent grant committee to evaluate and
-   publicly recommend Major Grant proposals, based on the committee's
-   expertise and the above inputs.
+6. Major Grants awarding is subject to approval by a five-seat Major Grant
+   Review Committee. The Major Grant Review Committee will be selected by the
+   ZF's Community Advisory Panel. The Major Grant Review Committee's funding
```

Comment:
It's now called "Community Governance Panel", no? (That's the term used in https://github.com/ZcashFoundation/Elections).

Reply:
Previously called Community Governance Panel, now Community Advisory Panel, but perhaps we should switch back given its commensurate increase in authority?

Reply:
Let's cross that bridge when we come to it, and stick with Community Advisory Panel for now.